host_vars/staging.righttoknow.org.au.yml

```diff
@@ -0,0 +1,16 @@
+# Host-specific variables for staging.righttoknow.org.au
+# Override connection settings for fresh EC2 instance
+
+# Use ubuntu user for initial connection
+ansible_user: ubuntu
+ansible_ssh_private_key_file: "{{ playbook_dir }}/terraform.pem"
+
+# Staging-specific settings
+name: staging.righttoknow.org.au
+rails_env: production
+domain: staging.righttoknow.org.au
+test_domain: staging-test.righttoknow.org.au
+
+# Disable cron and backups for staging
+cron_enabled: false
```

Comment:
Cron still needs to run in staging for some Alaveteli tasks

Reply:
we should build with cron off, so it doesn't pull from mailboxes right away